blockchain/blockchain_test.go

```diff
@@ -1438,3 +1441,121 @@ func TestCheckBlockChainVersion(t *testing.T) {
 	assert.Error(t, CheckBlockChainVersion(memDB))
 	assert.Equal(t, uint64(BlockChainVersion+1), *memDB.ReadDatabaseVersion())
 }
+
+var testCode = "60806040526101ca806100136000396000f300608060405260043610610057576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff16806312065fe01461005c578063b4ebea9914610087578063b6b55f25146100da575b600080fd5b34801561006857600080fd5b506100716100fa565b6040518082815260200191505060405180910390f35b34801561009357600080fd5b506100d8600480360381019080803563ffffffff169060200190929190803573ffffffffffffffffffffffffffffffffffffffff169060200190929190505050610119565b005b6100f86004803603810190808035906020019092919050505061018d565b005b60003073ffffffffffffffffffffffffffffffffffffffff1631905090565b60008090505b8263ffffffff168163ffffffff161015610188578173ffffffffffffffffffffffffffffffffffffffff166108fc60019081150290604051600060405180830381858888f1935050505015801561017a573d6000803e3d6000fd5b50808060010191505061011f565b505050565b803414151561019b57600080fd5b505600a165627a7a72305820ee29ede1bae5608d4c660a11196b4846247aedee2657207fd39c7d736882e1400029"
```

Comment:
Please add the original solidity code which generated this byte code.

Reply:
Applied. b7ae95f9483d9dc4675e66861ce5301ff5e8ffa3